### DISCUSSION TITLE
badware ? gpu.userbenchmark.com

### DISCUSSION THREAD

**Discussion by afkvido** [uBlockOrigin/uAssets#15540] [Q&A]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

`https://gpu.userbenchmark.com/`

### Describe the issue

`https://gpu.userbenchmark.com/` makes you download an exe with a virus

tested it on a library computer and that one is now in repairs

### uBlock Origin version

1.44.4 Latest

### Browser name and version

Google Chrome Version 107.0.5304.88 (Official Build) (64-bit)

### Settings

None

### Notes

I'm not sure where the badware list is but i found this virus

**Comment by mapx-** [comment]
What exactly did you download ?

**Comment by afkvido** [comment]
An exe that hijacked the library computer and triggered many antivirus warnings, then the screen started spinning around with some "GPU Tester" which didn't do any good things.

Had to plug it out.

**Comment by mapx-** [comment]
Just provide the name of that exe.

**Comment by afkvido** [comment]
```txt
UserBenchmarkInstaller.exe
```

**Comment by mapx-** [comment]
VirusTotal =>
- The site it's ok
- that file is flagged as malicious only by some minor vendor

![image](https://user-images.githubusercontent.com/10303732/200102425-8925c4db-97a9-4027-a726-233b3e62d3ea.png)
  
https://www.virustotal.com/gui/file/ed3a4d2c00348ee99205c7d2f1c69405b0ab046ce10566c7d74ba0e190b75ed6

**Reply by afkvido** [reply]
A VM could demonstrate why it's a virus.

Or i could visit a local library tomorrow and make a recording on my phone of what happens.

Sorry that this issue is poorly made and a bit insufficient. I'm just unsure on how to properly do this.

**Comment by ghost** [comment]
https://www.virustotal.com/gui/file/ed3a4d2c00348ee99205c7d2f1c69405b0ab046ce10566c7d74ba0e190b75ed6/community

**Reply by afkvido** [reply]
Never seen an account be deleted so quickly
